lib/heroku/command/pg_backups.rb

```diff
@@ -513,12 +513,15 @@ def unschedule_backups
     db = shift_argument
     validate_arguments!
 
+    if db.nil?
+      abort("Must specify database to unschedule backups for")
+    end
```

Comment:
I think it does make sense to require an argument. Maybe instead of pointing people to `help`, we should just list what's available to unschedule instead?